libs/domain/search/facet/facet.component.spec.ts

```diff
@@ -223,6 +259,39 @@ describe('SearchFacetComponent', () => {
         });
       });
     });
+
+    describe('enableClear', () => {
+      describe('is provided', () => {
```

Comment:
Description seems to be wrong. `When enableClear is false`?

Reply:
The description is correct. We are providing the `false` value for `enableClear` because it is `true` by default
```
//facet.component.ts
@signalProperty({ type: Boolean }) enableClear = true;